### DISCUSSION TITLE
Stabilization data missing

### DISCUSSION THREAD

**Discussion by AOtarolaR** [gopro/labs#1097] [Q&A]
My GoproMAX stops recording stabilization data at about 10:30 into the first video (in a long session)

Most of my work has always been longer than 45 minutes at a time and I've never had any problems until now.

When checking the storage folder you notice that the camera creates the .THM file only from the first video (and the .360 file can only be converted to .mov until minute 10:30).

The latest version of Gopro Labs is being used (v2.02.70, July 2024)

![goproMAX1](https://github.com/user-attachments/assets/120efde2-7b14-4a37-995d-1635c7f05cf5)
![goproMAX2](https://github.com/user-attachments/assets/f9e727b9-dfd4-4791-8b9e-a1e6a07426a7)
![goproMAX3](https://github.com/user-attachments/assets/3fe10a80-c012-4574-809b-1c6e74f9888c)

**Comment by dnewman-gpsw** [comment]
This does not seem related to Labs, what extensions are you running? As with Labs extensions (BITR, EXPT, etc.) the camera behavior is stock.  This is only one .THM per capture, so that is normal.

Are saying it never captures long any more?  That media looks old, update those.
